.github/workflows/lint-format.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           python-version: '3.12'
       - name: Install wpiformat
-        run: pip3 install wpiformat==2024.45
+        run: pip3 install --break-system-packages wpiformat==2024.45
```

Comment:
We use an RLI of this for frc-docs, so I'd prefer to set up a venv, rather then say --break-system-packages

https://docs.wpilib.org/en/stable/docs/software/advanced-gradlerio/code-formatting.html#wpiformat